### PR TITLE
Document the avifInputHasRemainingData() function

### PR DESCRIPTION
**Pull request by wantehchang**
I did not know what the function was for until I saw how it was used.

### CODE REVIEW

File: apps/avifenc.c [hunk 1]
```diff
@@ -188,6 +188,8 @@ static avifInputFile * avifInputGetNextFile(avifInput * input)
     }
     return &input->files[input->fileIndex];
 }
+
+// Returns whether the next avifInputGetNextFile() call will succeed.
```
Comment (wantehchang) [inline apps/avifenc.c:192]: Renaming this function something like "avifInputHasMoreFiles" would also be helpful by making the name more self-explanatory.

Comment (joedrago) [reply apps/avifenc.c:192]: "HasMoreFiles" is slightly incorrect, as this function will return true while you are still reading out of the middle of a multiframe y4m.

Comment (joedrago) [reply apps/avifenc.c:192]: I see why you chose the name you chose. Perhaps if anything, `avifInputGetNextFile()` needs a new name.

Comment (wantehchang) [reply apps/avifenc.c:192]: Thanks for taking a look! Note that this pull request does not rename the function. It only adds a comment.

**Pull request closed without merging**